### PR TITLE
Turkish Language Files and changelog updated.

### PR DESCRIPTION
**Pull request by serkanalgur**
:globe_with_meridians: Turkish Language files and changelog according to #138 & #170

### CODE REVIEW

File: src/_locales/tr/messages.json [hunk 1]
```diff
@@ -312,6 +328,10 @@
     "description": "This is an option shown in the add-on settings. It is one option for selecting the QR code size.",
     "hash": "280612358cfcc5357399f75b697a07bb"
   },
+  "subtitleMisc": {
+    "message": "Miscellaneous settings",
```
Comment (rugk) [inline src/_locales/tr/messages.json]: Ehm, this is not translated(?)…

File: src/_locales/tr/messages.json [hunk 2]
```diff
@@ -232,6 +236,18 @@
     "description": "The message, which appears, when settings are pre-defined (as managed options) by administrators.",
     "hash": "148604ebbeb7df28974d8233f1418d1f"
   },
+  "titleGeneral": {
+    "message": "Eklenti",
+    "description": "The title for the general settings group."
+  },
+  "titleQrCodeSetting": {
+    "message": "QR code",
```
Comment (rugk) [inline src/_locales/tr/messages.json]: QR code = QR code, I suspect. (or missing translation?)

**Comment by rugk** [comment]
(Hint: https://github.com/rugk/offline-qr-code/commit/89bbb19c22352832addedd83f9888147dd72788b :smiley: )

**Comment by serkanalgur** [comment]
Ok, i will change my commit.

File: assets/texts/en/Changelog/next.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 * **New:** The QR code now shows a warning if you try to enter too much text. ([#168](https://github.com/rugk/offline-qr-code/pull/168), thanks [@ongspxm](https://github.com/ongspxm))
+* **Improved:** The options page has been revamped and fits better into the style of the Photon options. ([#170](https://github.com/rugk/offline-qr-code/pull/170), thanks [@ongspxm](https://github.com/ongspxm))
```
Comment (rugk) [inline assets/texts/en/Changelog/next.md]: It seems you copied & pasted the changes here.
Note for the next time: Better merge my updated master into your branch, i.e. `git merge master` (after you have updated your master to get to the same state as my master.

Anyway, does not matter right now. :smile:

Comment (serkanalgur) [reply assets/texts/en/Changelog/next.md]: Yes, I did :) Alright captain. I will follow the rules next time :+1:

Comment (rugk) [reply assets/texts/en/Changelog/next.md]: Well, it's just a best practice for git in general… :smile:

**Pull request merged**